packages/squarev2-integration/src/__snapshots__/SquareV2PaymentMethod.spec.tsx.snap

```diff
@@ -3,41 +3,37 @@
 exports[`SquareV2 payment method should render placeholder form fields 1`] = `
 <div
   data-test="squarev2_placeholder_form"
```

Comment:
Do we still need this, since the test has moved to RTL?

Reply:
We still use this id in one of the RTL tests